### ISSUE TITLE
Not Responsive design 

### ISSUE THREAD

**Issue by NitinChakrawarti** [Omkar-Sonawane-23/Tourism#215]
**Describe the bug**
Website is not responsive in different device it losses it all responsiveness in mobile devices and small devices.

**To Reproduce**
Just open in mobile view 

**Expected behavior**
All elements of website needed to adapt as per screen size.

**Screenshots**
![Screenshot 2024-05-14 204831](https://github.com/Omkar-Sonawane-23/Tourism/assets/149225785/22b52286-7ba0-4477-b8af-39f937ae2178)

![Screenshot 2024-05-14 205120](https://github.com/Omkar-Sonawane-23/Tourism/assets/149225785/ce29e1d7-27d5-405d-b474-c342c380cfef)

**Additional context**
basically  we have to make website responsive.

@Omkar-Sonawane-23  please assign this is to me i want to work on this issue under GSSoC'24

Thank you.

**Comment by anshdeep0504** [comment]
Hey, allow me to work on this under GSSOC' 2024

**Comment by sameer-soni** [comment]
@NitinChakrawarti assigning this issue to you. Make the website responsive and don't forget to sync your local repo.

**Comment by NitinChakrawarti** [comment]
@sameer-soni  thanks for assigning, I will do it as soon as possible

**Comment by mtg718** [comment]
Hi @sameer-soni , please assign this issue to me under GSSoc'24

**Comment by NitinChakrawarti** [comment]
@sameer-soni as GSSoC'24 guidline add label for gssoc also

**Comment by zerotwoadarsh** [comment]
I would like to work on this issue. Can you assign it to me?
